server/web_ui/admin/pkg/README.md

```diff
@@ -0,0 +1,144 @@
+# Kanidm - Simple and Secure Identity Management
```

Comment:
Should have a "admin specific readme"

Reply:
Nah, we don't need one at all, we used to have to have README and LICENSE files for the WASM thing to build at all else it'd get sad.